extensions/wikidata/tests/src/org/openrefine/wikidata/schema/WbItemDocumentExprTest.java

```diff
@@ -92,11 +92,9 @@ public void testSerialize() {
         JacksonSerializationTest.canonicalSerialization(WbItemDocumentExpr.class, expr, jsonRepresentation);
     }
 
-    @Test
+    @Test(expectedExceptions = UnsupportedOperationException.class)
     public void testUnmodifiableList() {
-        List<WbNameDescExpr> nameDescExprs = expr.getNameDescs();
-        Assert.assertTrue(nameDescExprs.getClass().getName().contains("Unmodifiable"));
-        List<WbStatementGroupExpr> statementGroupExprs = expr.getStatementGroups();
-        Assert.assertTrue(statementGroupExprs.getClass().getName().contains("Unmodifiable"));
+        expr.getNameDescs().clear();
+        expr.getStatementGroups().clear();
```

Comment:
This will only test if `expr.getNameDescs()` is immutable, since that will already throw an exception. The second line will not be executed if the first line throws an exception. So you need to split that into two tests.